src/ops/ops.go

```diff
@@ -207,3 +211,48 @@ func (o *ops) renderControllerCm() error {
 	}
 	return nil
 }
+
+func (o *ops) GetVGByPV(pvName string) (string, error) {
```

Comment:
can we have some descriptions for those commands?

Reply:
we can, but this will result in a reduced job security